src/chrome/content/rules/lebsanft.org.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="lebsanft.org">
+  <target host="lebsanft.org" />
+  <target host="*.lebsanft.org" />
+  <securecookie host="^www\.lebsanft\.org$" name=".*" />
+  <exclusion pattern="^http://physik\.lebsanft\.org/"/>
+
+  <test url="http://lebsanft.org/" />
+  <test url="http://physik.lebsanft.org/" />
+  <test url="http://www.lebsanft.org/" />
+
+  <rule from="^http://lebsanft\.org/" to="https://lebsanft.org/"/>
+  <rule from="^http://www\.lebsanft\.org/" to="https://www.lebsanft.org/"/>
```

Comment:
You should just use the updated/simplified rule format here

```
<rule from="^http:" to="https:" />
```

You could then just explicitly only target www and no subdomain, and remove the exclusion. Simplies the ruleset somewhat